### PR TITLE
Level 1/task 6

### PR DESCRIPTION
**Pull request by PolinaXox**
task_6 v1.0

### CODE REVIEW

File: 1. Basics/task_6/hw6_counter.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ */
+define("FILE_NAME", "counter.txt");
+
+/**
+ * Counter start value
+ */
+define("COUNTER_START_VALUE", 0);
+
+if(!file_exists(FILE_NAME)) {
+    $file = fopen(FILE_NAME, "w") or die("I can't create a file");
```
Comment (anton-moroz) [inline 1. Basics/task_6/hw6_counter.php:13]: better not to use `or`, such behavior can confuse other developers

Comment (anton-moroz) [reply 1. Basics/task_6/hw6_counter.php:13]: you can use just `file_put_contents`

**Review by anton-moroz — CHANGES_REQUESTED** [review]

File: 1. Basics/task_6/hw6_counter.php (hunk 1 above)
Comment (PolinaXox) [reply 1. Basics/task_6/hw6_counter.php]: 👌 Done.

File: 1. Basics/task_6/hw6_counter.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+define("COUNTER_START_VALUE", 0);
+
+if(!file_exists(FILE_NAME)) {
+    file_put_contents(FILE_NAME, COUNTER_START_VALUE);
+}
+
+$counter = file_get_contents(FILE_NAME);
+echo $counter;
+$counter++;
+file_put_contents(FILE_NAME, $counter);
```
Comment (anton-moroz) [inline 1. Basics/task_6/hw6_counter.php:19]: `++$counter`

**Review by anton-moroz — APPROVED** [review]

File: 1. Basics/task_6/hw6_counter.php (hunk 2 above)
Comment (PolinaXox) [reply 1. Basics/task_6/hw6_counter.php]: Done. My program's become shoter 😃

**Pull request merged**